bofire/strategies/doe/design.py

```diff
@@ -402,6 +402,33 @@ def find_local_max_ipopt(
         )
         raise e
 
+    # determine number of experiments (only relevant if n_experiments is not provided by the user)
+    n_experiments = get_n_experiments(
+        domain=domain, model_type=model_type, n_experiments=n_experiments
+    )
+
+    if partially_fixed_experiments is not None:
+        # check if partially fixed experiments are valid
+        check_partially_fixed_experiments(
+            domain, n_experiments, partially_fixed_experiments
+        )
+        # no columns from partially fixed experiments which are not in the domain
+        partially_fixed_experiments = partially_fixed_experiments[
```

Comment:
One could also set `partially_fixed_experiments` as the return value of the check method. We do the same in domain for validate_experiments and validate_candidates.

Reply:
this does not seem to project out columns not in domain.inputs.